app/models/category.rb

```diff
@@ -2,3 +2,7 @@ class Category < ActiveRecord::Base
   validates :name, presence: true, uniqueness: true
   has_many :ideas
 end
+
+
+
+# I need to add a commment
```

Comment:
This PR is not really for any new code. I can't see the changes you made. Good work putting up a PR in general, but try to get my one that has some code I can review.